docs/user/aws/installconfig.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$id": "https://raw.githubusercontent.com/openshift/installer/master/docs/user/aws/installconfig.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "definitions": {
+    "AWSMachinePool": {
+      "additionalProperties": false,
+      "description": "OpenStack-specific properties to the OpenShift machine-pool.",
```

Comment:
Nit: "AWS specific properties to the OpenShift machine-pool."

Reply:
indeed!